src/components/pushClient/Banner.jsx

```diff
@@ -81,9 +81,8 @@ class BannerClient extends Component {
           onClick={e => {
             this.markAsSeen('close')
           }}
-        >
-          <Icon icon="cross" width="24" height="24" />
-        </Button>
+          icon="cross"
```

Comment:
In this case, wouldn't it be best to write `icon={<Icon icon="cross" width="24" height="24" />}`?
Because we're losing the size there.